week3/index.js

```diff
@@ -1,15 +1,77 @@
-const Express = require('express')
-const bodyParser = require('body-parser')
+"use strict"
+const express = require("express");
+const app = express();
+const bodyParser = require("body-parser");
+const fs = require("fs");
```

Comment:
You probably don't need to import `fs` here, since it's not used here (that I can see). You only need to `require` a library in the file where it gets used.